app/exporters/object_exporter.rb

```diff
@@ -8,7 +8,8 @@ class << self
     def export_lines
       Enumerator.new do |enum|
         export(klass, formatted_fields, default_fields) do |line|
-          enum << line + "\n"
+          # Encode to ISO-8859-1 because Excel can't handle UTF-8 for some reason.
+          enum << line.encode('ISO-8859-1', 'UTF-8', invalid: :replace, undef: :replace) + "\n"
```

Comment:
heeft het nog nut om een spec toe te voegen die daadwerkelijk kijkt of het ook goed gaat met ü etc? Lastig te testen misschien omdat het alleen in excel fout gaat.

Reply:
Goeie. Ik heb een spec toegevoegd.